joblib/memory.py

```diff
@@ -527,6 +540,21 @@ def _cached_call(self, args, kwargs):
                 shutil.rmtree(output_dir, ignore_errors=True)
                 out, metadata = self.call(*args, **kwargs)
                 argument_hash = None
+        else:
+            # FIXME: The statements below should be try/excepted
```

Comment:
I'm unsure about this. The comment is in the original code that I modified and I didn't want to loose it because it's not clear to me how important it is so I moved it along with the block it was referring to. I can dig more deeply if you like but my goal here was to keep my changes minimal since I'm a tourist in this code base.